docs/experimental-features/user-risk-score.asciidoc

```diff
@@ -0,0 +1,74 @@
+[[user-risk-score]]
+== User risk score
+
+NOTE: This feature is available for {stack} versions 8.3.0 and newer.
+
+The user risk score feature highlights risky usernames from within your environment. It utilizes a transform with a scripted metric aggregation to calculate user risk scores based on alerts  generated within the past 90 days. The transform runs hourly to update the score as new alerts are generated.
+
+Each alert's contribution to the user risk score is based on the alert's risk score (`signal.rule.risk_score`). The risk score is calculated using a weighted sum where rules with higher time-corrected risk scores also have higher weights. Each risk score is normalized on a scale of 0 to 100.
+
+The following table shows how risk levels are applied to a username, based on the normalized risk score:
+
+[width="100%",options="header"]
+|==============================================
+|Risk level |User risk score
+
+|Unknown |< 20
+|Low |20-40
+|Moderate |40-70
+|High     | 70-90
+|Critical  | > 90
+
+|==============================================
+
+[discrete]
+[[deploy-user-risk-score]]
+=== Deploy the user risk score package
+
+To deploy the user risk score framework in your environment, follow https://github.com/elastic/detection-rules/blob/main/docs/experimental-machine-learning/user-risk-score.md[these steps]. These instructions also include steps to enable the `riskyUsersEnabled` feature flag.
```

Comment:
It's a little awkward to mention the feature flag here, then mention it again two paragraphs/notes later. Could all that be combined here? Or maybe remove the second sentence here and keep the 2nd note below, since the note also points to the same instructions?